subgraph.polygon.local.yaml

```diff
@@ -40,8 +40,8 @@ dataSources:
           handler: handleBuy
         - event: PriceChange(indexed uint256,uint256)
           handler: handlePriceChange
-        # - event: Foreclosure(indexed address)
-        #   handler: handleForeclosure
+        - event: Foreclosure(indexed address,uint256)
+          handler: handleForeclosure
```

Comment:
Cool, if this works, you'll need to re-add these to the other versions of the graph.

Do you have permission to deploy the hosted graphs? (on thegraph.com)

Reply:
Currently it works but is unnecessary, will have that event included eventually though

Reply:
Don't have permission